a3p-integration/proposals/z:acceptance/vaults.test.js

```diff
@@ -0,0 +1,201 @@
+import test from 'ava';
+import {
+  agops,
+  bankSend,
+  getUser,
+  openVault,
+  adjustVault,
+  closeVault,
+  getISTBalance,
+  queryVstorage,
+  ATOM_DENOM,
+  USER1ADDR,
+  waitForBlock,
+} from '@agoric/synthetic-chain';
+
+const getFormattedVaultData = vaultData => {
+  const { values } = JSON.parse(vaultData.value);
+  const { body } = JSON.parse(values[0]);
+  const cleanedBody = JSON.parse(body.slice(1));
+  return cleanedBody;
+};
```

Comment:
Thank you for bringing this to my attention.
I have replaced the `queryVstorage` method with `getContractInfo`, and removed the `getFormattedVaultData` function since it is no longer required.
Change addressed on commit [641c9cd](https://github.com/Agoric/agoric-sdk/pull/10123/commits/641c9cd64443e3b3b63d0b8bfbb0a8b6279bc551)